tests/unit/YourCompany.spec.ts

```diff
@@ -15,12 +15,27 @@ import OfficeAddresses from '@/components/common/YourCompany/OfficeAddresses.vue
 import YourCompany from '@/components/common/YourCompany/YourCompany.vue'
 import { BenefitCompanyStatementResource } from '@/resources/Correction/BenefitCompanyStatementResource'
 import { BenefitCompanyResource } from '@/resources/Alteration/BenefitCompanyResource'
+import { SoleProprietorshipResource } from '@/resources/Correction/SoleProprietorshipResource'
 
 Vue.use(Vuetify)
 
 const localVue = createLocalVue()
 const vuetify = new Vuetify({})
 
+const flagsCompanyInfo = {
+  isValidCompanyName: false,
+  isValidBusinessType: false,
+  isValidNameTranslation: false,
+  isValidNatureOfBusiness: false,
+  isValidAddress: false,
+  isValidContactInfo: false,
+  isValidFolioInfo: false,
+  isValidOrgPersons: false,
+  isValidShareStructure: false,
+  isValidCompanyProvisions: false,
+  isValidResolutionDate: false
+}
```

Comment:
Ah, this is the old architecture. It will be nice when we get rid of this. See https://github.com/bcgov/entity/issues/13023

Reply:
For sure.